### PR TITLE
Added CoDICE dependencies

### PR DESCRIPTION
**Pull request by bourque**
This PR adds the CoDICE L0, L1a, and L1b dependencies

### CODE REVIEW

File: sds_data_manager/lambda_code/SDSCode/dependency_config_codice.py [hunk 1]
```diff
@@ -0,0 +1,383 @@
...
+        dependent_data_level="l1a",
+        dependent_descriptor="all",
+        relationship="HARD",
+        direction="DOWNSTREAM",
+    ),
+    # l1a to l1b
+    PreProcessingDependency(
+        primary_instrument="codice",
+        primary_data_level="l1a",
+        primary_descriptor="hskp",
```
Comment (tech3371) [inline sds_data_manager/lambda_code/SDSCode/dependency_config_codice.py:191]: It's ok right now. Something we can bring up during nomenclature discussion in the future. Should we use `hk` or `hskp`?

**Review by tech3371 — APPROVED** [review]
It looks good to me. Nice

File: sds_data_manager/lambda_code/SDSCode/dependency_config_codice.py (hunk 1 above)
Comment (bourque) [reply sds_data_manager/lambda_code/SDSCode/dependency_config_codice.py:191]: Yeah, I agree, something we should discuss. I know that CoDICE uses `hskp` all throughout their algorithm document ...

**Comment by bourque** [comment]
Just noting here that I need to change this to have a one-to-one mapping of L0 APIDs/descriptors to L1a products

**Review by laspsandoval — APPROVED** [review]
Nice work.

**Review by tech3371 — COMMENTED** [review]
looks good to me. Post SIT-3, we can figure out how we want data from POC and then change how we want to trigger l1a process based on outcome from that discussion.

**Pull request merged**